ssz-rs/src/merkleization/proofs.rs

```diff
@@ -295,6 +295,34 @@ pub(crate) mod tests {
         assert!(result.is_ok());
     }
 
+    #[test]
```

Comment:
nice! I may refactor this into a Cargo benchmark, but this is fine for now

Reply:
Yeah I'm also experimenting with hashtree_rs, could add benchmarks in a follow-up PR!